css-conditional-5/font-tech-format-explainer.md

```diff
@@ -0,0 +1,87 @@
+# `font-tech()` and `font-format()` `<supports-feature>` Extensions to CSS `@supports`
+
+## Authors
+
+* Dominik Röttsches, [@drott](drott@chromium.org)
+* Chris Lilley, [@svgeesus](chris@w3.org)
+* Jonathan Kew, [@jfkthame](jfkthame@gmail.com)
+
+## Introduction
+
+`font-tech()` and `font-format()` are extensions to the CSS Conditional Syntax
+which enable declarative syntax to use a different set of style rules depending
+on font format support, as well as programmatic feature detection of
+capabilities of the font stack. The latter covers a lack of functionality in web
+platform APIs, as most JS APIs and DOM functionality can be easily feature
+detected, but the abilities of the font stack could not be queried in that way
+so far.
+
+The proposed syntax was introduced following discussion in [TAG review
+#666](https://github.com/w3ctag/design-reviews/issues/666) on a previous syntax
+for the `src:` descriptor, and subsequent [resolutions in the CSS
+WG](https://github.com/w3c/csswg-drafts/issues/6520#issuecomment-947810568) to
+harmonize functionality between the `src:` descriptor parsing, and CSS
```

Comment:
Maybe it's worth saying explicitly whether the syntax that goes inside of these `@supports` functions is the same as the syntax that goes inside the corresponding `src` descriptor functions?  (I don't know the answer off the top of my head.)

Reply:
Its certainly worth saying why it is different :) because the one in `src` on `@font-face` is clearly about a font so we made the name shorter, like `tech()` while the one inside `@supports` lacks that context so the name is `font-tech()` there. The spec explicitly equates the two and the functions take the same values in both places.

Reply:
Brief syntax comparison added and explaining why `font-` prefix is not used in `@font-face`.